classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -162,12 +166,84 @@ public function render() {
 						<?php endif; ?>
 						<a class="copy-curl" data-request="<?php echo esc_attr( addcslashes( $curl_request, '"' ) ); ?>">Copy cURL Request</a>
 					</li><?php
-					
+
 				endforeach;
 
 			?></ol><?php
 
 		endif;
 	}
 
+	/**
+	 * Display debug info that effects the Elastic Press logs being visible or not.
+	 */
+	protected function display_debugging_info() {
+		$is_debug_errors = false;
+		?>
+		<div class="qm-boxed">
+			<section>
+				<h3><?php esc_html_e( 'Debug Info', 'debug-bar' ); ?></h3>
+				<p>
+					<?php esc_html_e( 'Elastic Press query logs not being displayed can be due to one of the following reasons:', 'debug-bar' ); ?>
+				</p>
+				<table>
+					<tbody>
+						<?php
+						$is_wp_debug = defined( 'WP_DEBUG' ) && WP_DEBUG;
+						$is_wp_ep_debug = defined( 'WP_EP_DEBUG' ) && WP_EP_DEBUG;
+
+						// Display error message only if both WP_DEBUG and WP_EP_DEBUG are turned off.
+						if ( ! ( $is_wp_debug || $is_wp_ep_debug ) ) :
```

Comment:
It's quite hard to understand this as it's written. Can you make it just `if ( ! $is_wp_debug && ! $is_wp_ep_debug )`?

Reply:
mais bien sur! Nice one, pity I didn't come up with the simplicity of it :(